schedule/yam/agama_unattended_keyboard_layout.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: agama_unattended_keyboard_layout
+description: >
+  Perform unattended installation with Agama, with keyboard and language changed.
+schedule:
+  - yam/agama/boot_agama
+  - yam/agama/agama_auto
+  - installation/grub_test
+  - installation/first_boot
+  - yam/validate/validate_product
+  - yam/validate/validate_first_user
```

Comment:
these two test are not needed, we don't need to repeat this in each installation, only in the default.

Reply:
Yes, I removed these and rerunning the VRs, will see whether the layout is still different with SLE15SP7.